doc/api/path.markdown

```diff
@@ -62,12 +63,14 @@ path.dirname('/foo/bar/baz/asdf/quux')
 '/foo/bar/baz/asdf'
 ```
 
-## path.extname(p)
+## path.extname(path)
 
 Return the extension of the path, from the last '.' to end of string
 in the last portion of the path.  If there is no '.' in the last portion
 of the path or the first character of it is '.', then it returns
-an empty string.  Examples:
+an empty string. `path` must be a string.
```

Comment:
Yeah, I realized we're pretty inconsistent with argument description all over, so disregard my comment.